parser/ast_nodes.js

```diff
@@ -65,9 +65,34 @@ function Comment(string) {
 Comment.prototype.type = ast_types.comment;
 /* -------- */
 
+/* Cleave Node */
+function Cleave(key) {
+    Tag.call(this, key);
+};
+
+Cleave.prototype = Object.create(Tag.prototype);
+
+Cleave.prototype.type = ast_types.cleave;
+
+Cleave.prototype.addChild = function(child) {
+    if(!(child instanceof Tag || child instanceof Text || child instanceof Comment)) {       
```

Comment:
Why can't a Cleave tag have a child of tag, text, or comment?

Reply:
It can only have those types. If it doesn't, it throws an error

Reply:
I read that wrong my bad!